script/output/mainnet/M1_deployment_mainnet_2023_6_9.json

```diff
@@ -1,6 +1,9 @@
 {
   "addresses": {
+    "avsDirectory": "0x0000000000000000000000000000000000000000",
+    "avsDirectoryImplementation": "0x0000000000000000000000000000000000000000",
     "baseStrategyImplementation": "0xdfdA04f980bE6A64E3607c95Ca26012Ab9aA46d3",
+    "beaconOracleAddress": "0x0000000000000000000000000000000000000000",
```

Comment:
should this be non-zero? I get that the `avsDirectory` is set to the zero address because it is not deployed yet.
I would guess that this doesn't matter, just searching for anything, honestly.

Reply:
Ah yes I added this because the `_parseDeployedContracts` helper will throw an error when trying to parse it out. It was used for the holesky deployment so I added these 0 addresses here based on the current mainnet state.
The beaconOracle address gets overwritten when `_parseInitialDeploymentParams` is called right after.